improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -381,6 +382,14 @@ def process(self, *cubes: Union[Cube, CubeList]) -> Cube:
         return self._make_humidity_cube(humidity)
 
 
+def check_for_pressure_cube(cubes) -> str:
+    """Checks a list of cubes to see if any has the pressure in the name and returns the name as a string"""
```

Comment:
This will return the first cube with "pressure" in the name and ignore all the others. We probably ought to raise an Exception if multiple cubes match.